java/settings.gradle

```diff
@@ -62,6 +62,7 @@ include ':add-graphics-renderer',
         ':openstreetmap-layer',
         ':picture-marker-symbols',
         ':project',
+        ':query-map-image-sublayer',
```

Comment:
put in alphabetical order (it may have been originally, but new samples have reordered things)

Reply:
what do i put in alphabetical order? the recently added project or the whole file? To me the project looks in alphabetical order, maybe I am not seeing what you are. Thanks!

Reply:
just put query map image sublayer below portal user info :)